<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PatientDialog</class>
 <widget class="QDialog" name="PatientDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fiche patient</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>381</width>
     <height>462</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QLineEdit" name="patientNameLineEdit"/>
    </item>
    <item row="6" column="1">
     <widget class="QLabel" name="patientPriorityLabel">
      <property name="text">
       <string>Priorité</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLabel" name="patientConsultationDateLabel">
      <property name="text">
       <string>Jour de consultation</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="patientCityLabel">
      <property name="text">
       <string>Ville (*)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLineEdit" name="patientAddressLineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QDateEdit" name="patientConsultationDateDateEdit"/>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="patientIdLineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">background: rgb(220, 220, 220);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QSpinBox" name="patientConsultationDurationSpinBox">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>15</number>
      </property>
      <property name="maximum">
       <number>120</number>
      </property>
      <property name="singleStep">
       <number>15</number>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QSpinBox" name="patientPrioritySpinBox">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>5</number>
      </property>
      <property name="value">
       <number>1</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QLabel" name="patientIdLabel">
      <property name="text">
       <string>Numéro (ID)</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="patientZipCodeLabel">
      <property name="text">
       <string>Code postal (*)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="patientCityLineEdit"/>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="patientFirstnameLabel">
      <property name="text">
       <string>Prénom (*)</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="patientNameLabel">
      <property name="text">
       <string>Nom (*)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="patientFirstnameLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="patientZipCodeLineEdit"/>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="patientConsultationDurationLabel">
      <property name="text">
       <string>Durée de consultation (en minutes)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="patientAddressLabel">
      <property name="text">
       <string>Adresse (*)</string>
      </property>
     </widget>
    </item>
    <item row="12" column="0">
     <widget class="QLabel" name="patientAvailableResourcesLabel">
      <property name="text">
       <string>Ressources à allouer (selectionner)</string>
      </property>
     </widget>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="patientPhoneNumberLabel">
      <property name="text">
       <string>Téléphone</string>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLineEdit" name="patientPhoneNumberLineEdit"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="patientCommentaryLabel">
      <property name="text">
       <string>Commentaire</string>
      </property>
     </widget>
    </item>
    <item row="11" column="0" colspan="2">
     <widget class="QLineEdit" name="patientCommentaryLineEdit"/>
    </item>
    <item row="14" column="0">
     <widget class="QLabel" name="patientAllocatedResourcesLabel">
      <property name="text">
       <string>Ressources allouées</string>
      </property>
     </widget>
    </item>
    <item row="15" column="0" colspan="2">
     <widget class="QListView" name="patientAllocatedResourcesListView">
      <property name="styleSheet">
       <string notr="true">background: rgb(220, 220, 220);</string>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
     </widget>
    </item>
    <item row="13" column="0" colspan="2">
     <widget class="QListView" name="patientAvailableResourcesListView">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::MultiSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="uniformItemSizes">
       <bool>true</bool>
      </property>
      <property name="selectionRectVisible">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>530</y>
     <width>164</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="cancelPatientEditionButton">
      <property name="text">
       <string>Annuler</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="validatePatientEditionButton">
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="patientDialogHeader">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Fiche patient</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>patientNameLineEdit</tabstop>
  <tabstop>patientFirstnameLineEdit</tabstop>
  <tabstop>patientAddressLineEdit</tabstop>
  <tabstop>patientCityLineEdit</tabstop>
  <tabstop>patientZipCodeLineEdit</tabstop>
  <tabstop>patientConsultationDateDateEdit</tabstop>
  <tabstop>patientConsultationDurationSpinBox</tabstop>
  <tabstop>patientPrioritySpinBox</tabstop>
  <tabstop>patientPhoneNumberLineEdit</tabstop>
  <tabstop>patientIdLineEdit</tabstop>
  <tabstop>patientCommentaryLineEdit</tabstop>
  <tabstop>patientAvailableResourcesListView</tabstop>
  <tabstop>cancelPatientEditionButton</tabstop>
  <tabstop>validatePatientEditionButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
